internal/server/server_test.go

```diff
@@ -26,6 +26,10 @@ type mockApiClient struct {
 	TeamsForSelection  []model.Team
 }
 
+func (m mockApiClient) ReassignClientToCaseManager(context sirius.Context, i int, strings []string) (string, error) {
+	panic("implement me")
```

Comment:
What's the panic for?

Reply:
It was me coping the other pattern but I have change it now. It does not matter what it returns just that its on the mockApiClient